add Licenses View Controller

amirfl opened the pull request:
![screen shot 2018-07-03 at 8 48 06 pm](https://user-images.githubusercontent.com/364326/42250925-6ab23f9c-7f02-11e8-8115-b4dfeccb147a.png)


![screen shot 2018-07-03 at 8 46 28 pm](https://user-images.githubusercontent.com/364326/42250911-531f6062-7f02-11e8-85f8-cc7a8034aa47.png)

amirfl committed:
add LicensesViewController

WERUreo commented on PetAdoption-iOS/PetAdoption-iOS/LicensesViewController.swift:
```diff
@@ -0,0 +1,67 @@
+
+import UIKit
+
+struct item {
```
The only criticism I would have is that the convention is to capitalize classes and structs
> Whenever you define a new structure or class, you define a new Swift type. Give types UpperCamelCase names (such as SomeStructure and SomeClass here) to match the capitalization of standard Swift types (such as String, Int, and Bool). Give properties and methods lowerCamelCase names (such as frameRate and incrementCount) to differentiate them from type names.

> https://docs.swift.org/swift-book/LanguageGuide/ClassesAndStructures.html

Other than that, looks good!

amirfl committed:
rename item -> Item

amirfl committed:
add licenses

amirfl committed:
Adjust table view cell height by license detail text length

amirfl committed:
remove excessive tableview cell dividers by adding empty footer view

amirfl commented:
Thanks for the review.

Changed "item" to "Item". Also added a few more licenses, not sure if I got all of the 3rd licenses we are using.
Also refactored so that the detail text of each license fits in the table view cell.

![simulator screen shot - iphone 8 plus - 2018-07-05 at 18 02 47](https://user-images.githubusercontent.com/364326/42350259-abb7b7de-807d-11e8-9f3f-a219a9bc3139.png)

amirfl committed:
Refactor webviewer to a custom class.

Reason is that SFSafariViewControlelr shows its own navigation bar

WERUreo commented:
Looks good.  Thanks!
